nexus/types/src/external_api/params.rs

```diff
@@ -466,6 +466,28 @@ pub struct InstanceCreate {
     /// The disks to be created or attached for this instance.
     #[serde(default)]
     pub disks: Vec<InstanceDiskAttachment>,
+
+    /// Whether to not only create the Instance, but also run it.
+    ///
+    /// If `true`, the instance will be created but not started.
+    /// If `false`, the instance will be created and started immediately.
+    ///
+    /// By default, the instance will be created and started.
+    #[serde(default)]
+    pub only_create: bool,
```

Comment:
Hey @luqmana, how would you feel about `start_after_create` or `start` with a default of `true` or something along those lines? It's a minor thing, but `only_create` isn't clear (outside the documentation) what the implications of the flag are.

Reply:
Oh I'm very open to better naming all around! (in fact i'm not even sure I fully like Provisioned but just went with it to build the rest).

I did initially have it as `start` with a default of `true` but ran into some issues with the OpenAI spec and some of the generated bindings/structures. Adam did make a fix in progenitor so I can give it a try again.

Reply:
ok, flipped the flag and renamed it to `start`